packages/frontend/src/pages/InferenceServerDetails.svelte

```diff
@@ -74,6 +128,49 @@ $: service = $inferenceServers.find(server => server.container.containerId === c
                 </div>
               </div>
             </div>
+
+            <!-- code client -->
+            <div>
+              <div class="flex flex-row">
+                <span class="text-base grow">Client code</span>
+
+                <!-- language choice -->
+                <select
+                  required
+                  aria-label="snippet language selection"
+                  bind:value="{selectedLanguage}"
+                  on:change="{onLanguageChange}"
+                  id="languages"
+                  class="border text-sm rounded-lg focus:ring-purple-500 focus:border-purple-500 block p-1 bg-charcoal-900 border-charcoal-900 placeholder-gray-700 text-white"
+                  name="languages">
+                  {#each $snippetLanguages as language}
+                    <option class="my-1" value="{language.key}">{language.label}</option>
+                  {/each}
+                </select>
+                {#if selectedVariant !== undefined}
+                  <select
```

Comment:
The select should be disable if the language has only one variant

Reply:
Yes

Reply:
I would rather apply the suggestion of @jeffmaury as displaying `N/A` or `Not Applicable` could be weird, when we do have a name for the variant. Showing a hint, of the library used generally.

Here is why it look likes when it is disabled when only one variant.

<img width="159" alt="image" src="https://github.com/projectatomic/ai-studio/assets/42176370/5c2d5eb9-5960-49e9-9a04-916ab762587a">

<img width="140" alt="image" src="https://github.com/projectatomic/ai-studio/assets/42176370/386a9b61-3278-4a52-8c48-1157789771c6">

<img width="188" alt="image" src="https://github.com/projectatomic/ai-studio/assets/42176370/600d9a71-f035-4f84-b1fc-f71c8a845984">